packages/common/driver-definitions/src/storage.ts

```diff
@@ -198,6 +198,17 @@ export interface IDocumentStorageService extends Partial<IDisposable> {
 	 */
 	readBlob(id: string): Promise<ArrayBufferLike>;
 
+	/**
+	 * Builds a URL for accessing a blob by its storage ID.
+	 * @param storageId - The storage ID of the blob
+	 * @returns The URL to access the blob, or undefined if URL cannot be built
+	 * @remarks
+	 * This method leverages the driver's internal URL building logic to create
+	 * a direct access URL for the blob. The URL may expire and does not support
+	 * permalinks. This is intended for temporary integration scenarios only.
+	 */
+	buildBlobUrl?(storageId: string): string | undefined;
```

Comment:
This might not belong on the driver definition - consider whether it makes more sense either (A) on the url resolver or similar or (B) implemented only on the drivers that support it but not on the shared interface